include/dd/StochasticNoiseOperationTable.hpp

```diff
@@ -77,7 +65,7 @@ namespace dd {
 
     private:
         std::size_t                            nvars;
-        static constexpr auto                  opCount = static_cast<std::uint_fast8_t>(NoiseOperationKind::opCount);
+        static constexpr auto                  opCount = static_cast<std::uint_fast8_t>(42);
```

Comment:
This is a consequence since the used enum is defined within the QFR (OpType), which I cannot access here, although I need its size. Currently, the enum has 41 elements. The additional element is a buffer if a new operation should be added to the enum (also I liked the number ;)). 

When the OpType increases in size, this can cause problems. I caught this problem by adding asserts and comments, so it will be easy to fix.
About the variable name. I changed opCount to numberOfOperations

Reply:
I am not a big fan of having to touch the DD Package whenever I want to add a new operation to the QFR library.

I see two possible solutions:
 
Changing to `std::uint8_t` (accepting that there are platforms where this is not available) and using `std::numeric_limits<std::uint8_t>::max()` as the number of operations. This eliminates the magic number completely and leaves space for up to 255 operations, which should be plenty.
The downside is that it currently wastes `~ 200 * sizeof(Edge) * num_qubits` bytes of memory.
That should amount to a maximum of `200 * 24 Bytes * 128 = 0.6 MB`. 
I could live with that. if that is considered too much of an overhead, then maybe just fix the magic number to `64`.

A second solution would be to extract the number of operations into a template parameter of the noise table and consequently of the package itself. Then the magic number can be provided as a default value. Although, also here, I would argue that allowing for some headroom would be beneficial, i.e., setting the number to `64`.

In both settings, the DDPackage continues to work if new operations are added in the QFR.

Reply:
> Although, also here, I would argue that allowing for some headroom would be beneficial, i.e., setting the number to 64.
I changed `numberOfOperations` to 64